src/webapi/touch.rs

```diff
@@ -0,0 +1,138 @@
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/identifier)
+    #[inline]
+    pub fn identifier(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.identifier;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the X coordinate of the touch point relative to the left edge of the screen.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/screenX)
+    #[inline]
+    pub fn screen_x(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.screenX;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the Y coordinate of the touch point relative to the left edge of the screen.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/screenY)
+    #[inline]
+    pub fn screen_y(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.screenY;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the X coordinate of the touch point relative to the left edge of the browser viewport, not including any scroll offset.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/clientX)
+    #[inline]
+    pub fn client_x(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.clientX;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the Y coordinate of the touch point relative to the left edge of the browser viewport, not including any scroll offset.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/clientY)
+    #[inline]
+    pub fn client_y(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.clientY;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the X coordinate of the touch point relative to the left edge of the document. Unlike clientX, this value includes the horizontal scroll offset, if any.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/pageX)
+    #[inline]
+    pub fn page_x(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.pageX;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the Y coordinate of the touch point relative to the left edge of the document. Unlike clientX, this value includes the horizontal scroll offset, if any.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/pageY)
+    #[inline]
+    pub fn page_y(&self) -> f64 {
+        js!(
+            return @{self.as_ref()}.pageY;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the Element on which the touch point started when it was first placed on the surface, even if the touch point has since moved outside the interactive area of that element or even been removed from the document.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/target)
+    #[inline]
+    pub fn target(&self) -> Element {
```

Comment:
According to specs this returns `EventTarget`, not `Element`.

(If you really want to add a convenance method which returns an `Element` you can add `fn target_element( &self ) -> Option< Element >` if you want.)